presto-cassandra/src/main/java/com/facebook/presto/cassandra/Murmur3PartitionerTokenRing.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cassandra;
+
+import java.math.BigInteger;
+
+import static java.math.BigInteger.ONE;
+
+public final class Murmur3PartitionerTokenRing
+        implements TokenRing
+{
+    public static final Murmur3PartitionerTokenRing INSTANCE = new Murmur3PartitionerTokenRing();
+
+    private static final BigInteger MIN_TOKEN = BigInteger.valueOf(Long.MIN_VALUE).subtract(ONE);
```

Comment:
The Spark code has a different distance: https://github.com/datastax/spark-cassandra-connector/blob/75719dfe0e175b3e0bb1c06127ad4e6930c73ece/spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/partitioner/dht/TokenFactory.scala (Long.MIN_VALUE, not Long.MIN_VALUE - 1). Where'd you get these values?